packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -1045,35 +1022,53 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
             switch (summaryObject.type) {
                 case api.SummaryType.Tree: {
+                    treesPathToSummaryTreeCacheLatest.set(`.app${path}/${key}`, summaryObject);
```

Comment:
I think it makes more sense for this function to know nothing about .app, but rather the caller to pass path = .app

Reply:
Removed.